blocklist-processing-engine/src/config/config.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'path';
+import { config as dotenv } from 'dotenv';
+
+// Load .env file into process.env if it exists. This is convenient for running locally.
+dotenv({
+    path: path.resolve(__dirname, '../../.env')
+});
+
+const configuration: configObj = {
+    kafkaEndpoint: <string>process.env.KAFKA_ENDPOINT,
+    topics: (<any>process.env.TOPICS).split(','),
+    resultTopic: <string>process.env.RESULT_TOPIC,
+    partition: parseInt(<any>process.env.PARTITION),
```

Comment:
my previous implementation of eslint was wrong. I fixed it and this gets highlighted. Would probably be good if you take a look there.